core/tests/webdriverio_desktop/explorationHistoryTab.js

```diff
@@ -131,7 +131,7 @@ describe('Exploration history', function() {
       },
       10: {
         text: '  default_outcome:',
-        highlighted: false
+        highlighted: true
```

Comment:
Why does this change to `true`? In the screen recording, this line doesn't look highlighted

Reply:
I forgot to update the PR desc recording.. from the new package we are using from NPM, it is also highlighting the just parent element of the changed content. updated the screen recording

Reply:
Ahh okay, it looks like the NPM package is just creating a few, larger changed "chunks" than before (where it created more, smaller chunks). I think that's fine